travis-ymls/node-i18n-utilities.yml

```diff
@@ -0,0 +1,24 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : node-i18n-utilities
+# Source Repo         : https://github.com/IBM/node-i18n-utilities
+# Travis Job Link     : https://travis-ci.com/github/nageshlop/node-i18n-utilities/builds/213430520
+# Created travis.yml  : Yes
+# Maintainer          : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Script License      : Apache 2.0
+#
+# ----------------------------------------------------------------------------
+dist: xenial
+arch:
+   - amd64
+   - ppc64le  
+language: node_js
+node_js:
+   - "10"
```

Comment:
I see that the repo is not maintained so the latest node versions might not work but any particular reason to select version 10?

Reply:
@seth-priya I referred the previous travis builds and as the project is related to javascript added nodejs 10 and tested. Also verified with nodejs 15 version its working fine. Let me know how to go about this.
Build Log: https://travis-ci.com/github/nageshlop/node-i18n-utilities/builds/213742618

Reply:
I believe they have moved to another CI and that is why their travis builds may not be updated? In any case, would suggest adding the latest versions as well.